ui/src/containers/Layout.js

```diff
@@ -143,14 +128,44 @@ const Layout = props => {
     },
   ];
 
+  const filterLanguage = languages.find(lang => lang.name !== language);
```

Comment:
you should use `filter` instead of `find` here. In our case, we have 2 languages, that's why it works.